Makefile

```diff
@@ -1,3 +1,12 @@
+export GO111MODULE=on
+
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
+
+include ./make/*.mk
+.DEFAULT_GOAL := help
+
+
 #! /usr/bin/make
```

Comment:
I guess you can remove all those commented lines, now

Reply:
Already done. See e264d7d. You were just 30s too early or so :-)